### PR TITLE
fix: site-footer-edits-en

### PR DESCRIPTION
**Pull request by prycrane**
### This pull request is now being handled by:

- [Footer update (new version) #267](https://github.com/canada-ca/design-system/pull/267)

### Old preview - no longer updated!
This is a text edit.  I updated the pages to the new Jekyll template.  @delisma - let's have a look at how the h1 is handled.  Thanks.

- https://deploy-preview-222--design-system-canada-ca.netlify.app/common-design-patterns/site-footer.html
- https://deploy-preview-222--design-system-canada-ca.netlify.app/common-design-patterns/site-footer-main.html
- https://deploy-preview-222--design-system-canada-ca.netlify.app/common-design-patterns/site-footer-contextual.html
- https://deploy-preview-222--design-system-canada-ca.netlify.app/common-design-patterns/site-footer-sub.html

French pull request: https://github.com/canada-ca/systeme-conception/pull/131

### CODE REVIEW

File: common-design-patterns/site-footer-contextual.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        <li><a href="https://github.com/canada-ca/design-system-systeme-conception/issues">Discuss the pattern in GitHub
+          issues</a></li>
+        <li><a href="https://design-gc-conception.slack.com/join/shared_invite/enQtODE1OTc5Mzg5NzQ4LWQ3MjZjMTdjMjk2ZTZmMTJjYWQ3ZmRiNDYwYjRmN2NjYzQyNjFlNDBlY2FkNWE1ODg2YjExY2QwZmVjN2MwMGM">Join the conversation on Slack</a></li>
+        <li><a href="mailto:dto.btn@tbs-sct.gc.ca">Send an email to the Digital Transformation Office</a></li>
+      </ul>
+    </section>
+  </div>
+</div>
+
+<!-- START PAGE FEEDBACK WIDGET -->
```
Comment (delisma) [inline common-design-patterns/site-footer-contextual.md]: We should remove the feedback widget

File: common-design-patterns/site-footer-main.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+    link: "https://design.canada.ca/common-design-patterns/site-footer.html"
+date: 2022-11-30
+dateModified: 2023-04-06
+description: "The requirements listed in this design manual apply to departments and other portions of the federal public administration as set out in Schedules I, I.1 and II of the Financial Administration Act. As such, in-scope institutions must apply Canada.ca design requirements for all public-facing web sites or digital services."
+layout: default
+title: "Global footer: Main band"
+---
+<p><strong>Last updated</strong>: 2023-04-06</p>
+<div>
+  <div class="mwsgeneric-base-html parbase section">
```
Comment (delisma) [inline common-design-patterns/site-footer-main.md]: These classes are AEM specific.

File: common-design-patterns/site-footer-main.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+        <li><a href="https://github.com/canada-ca/design-system-systeme-conception/issues">Discuss the pattern in GitHub
+          issues</a></li>
+        <li><a href="https://design-gc-conception.slack.com/join/shared_invite/enQtODE1OTc5Mzg5NzQ4LWQ3MjZjMTdjMjk2ZTZmMTJjYWQ3ZmRiNDYwYjRmN2NjYzQyNjFlNDBlY2FkNWE1ODg2YjExY2QwZmVjN2MwMGM">Join the conversation on Slack</a></li>
+        <li><a href="mailto:dto.btn@tbs-sct.gc.ca">Send an email to the Digital Transformation Office</a></li>
+      </ul>
+    </section>
+  </div>
+</div>
+
+<!-- START PAGE FEEDBACK WIDGET -->
```
Comment (delisma) [inline common-design-patterns/site-footer-main.md]: With the default layout we have to remove the Page Feedback Widget.

File: common-design-patterns/site-footer-sub.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+    link: "https://design.canada.ca/common-design-patterns/site-footer.html"
+date: 2022-11-30
+dateModified: 2023-04-06
+description: "This is a mandatory element of the global (site-wide) footer."
+layout: default
+title: "Global footer: Sub-footer band"
+---
+<p><strong>Last updated</strong>: 2023-04-06</p>
+<div>
+  <div class="mwsgeneric-base-html parbase section">
```
Comment (delisma) [inline common-design-patterns/site-footer-sub.md]: These classes are specific to AEM

File: common-design-patterns/site-footer-sub.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+        <li><a href="https://github.com/canada-ca/design-system-systeme-conception/issues">Discuss the pattern in GitHub
+          issues</a></li>
+        <li><a href="https://design-gc-conception.slack.com/join/shared_invite/enQtODE1OTc5Mzg5NzQ4LWQ3MjZjMTdjMjk2ZTZmMTJjYWQ3ZmRiNDYwYjRmN2NjYzQyNjFlNDBlY2FkNWE1ODg2YjExY2QwZmVjN2MwMGM">Join the conversation on Slack</a></li>
+        <li><a href="mailto:dto.btn@tbs-sct.gc.ca">Send an email to the Digital Transformation Office</a></li>
+      </ul>
+    </section>
+  </div>
+</div>
+
+<!-- START PAGE FEEDBACK WIDGET -->
```
Comment (delisma) [inline common-design-patterns/site-footer-sub.md]: We should remove the Page Feedback Widget

File: common-design-patterns/site-footer.md [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+  <h2 id="discuss">Discussion</h2>
+  <ul>
+    <li><a href="https://github.com/canada-ca/design-system-systeme-conception/issues">Discuss the pattern in GitHub
+      issues</a></li>
+    <li><a href="https://design-gc-conception.slack.com/join/shared_invite/enQtODE1OTc5Mzg5NzQ4LWQ3MjZjMTdjMjk2ZTZmMTJjYWQ3ZmRiNDYwYjRmN2NjYzQyNjFlNDBlY2FkNWE1ODg2YjExY2QwZmVjN2MwMGM">Join the conversation on Slack</a></li>
+    <li><a href="mailto:dto.btn@tbs-sct.gc.ca">Send an email to the Digital Transformation Office</a></li>
+  </ul>
+</section>
+
+<!-- START PAGE FEEDBACK WIDGET -->
```
Comment (delisma) [inline common-design-patterns/site-footer.md]: We should remove the Page Feedback Widget

**Review by delisma — CHANGES_REQUESTED** [review]
We need to remove the Page Feedback Widget from these pages otherwise it will conflicts with the default layout.

File: common-design-patterns/site-footer-contextual.md (hunk 1 above)
Comment (prycrane) [reply common-design-patterns/site-footer-contextual.md]: Done.

File: common-design-patterns/site-footer-main.md (hunk 2 above)
Comment (prycrane) [reply common-design-patterns/site-footer-main.md]: Fixed.

File: common-design-patterns/site-footer-main.md (hunk 3 above)
Comment (prycrane) [reply common-design-patterns/site-footer-main.md]: Done

File: common-design-patterns/site-footer-sub.md (hunk 4 above)
Comment (prycrane) [reply common-design-patterns/site-footer-sub.md]: Fixed

File: common-design-patterns/site-footer-sub.md (hunk 5 above)
Comment (prycrane) [reply common-design-patterns/site-footer-sub.md]: Done

File: common-design-patterns/site-footer.md (hunk 6 above)
Comment (prycrane) [reply common-design-patterns/site-footer.md]: Done

File: common-design-patterns/site-footer.md
Comment (LPipes4) [inline common-design-patterns/site-footer.md:1]: Something happened with the preview of this one - only the code shows. Need to double check what's going on before we push it. The first 2 pages look fine and are ready to go. For this one, maybe it's just the preview that's broken but we need to have a look and see.

File: common-design-patterns/site-footer.md [hunk 7]
```diff
@@ -0,0 +1,340 @@
+---
```
Comment (LPipes4) [inline common-design-patterns/site-footer.md:1]: I tried putting this at the top of the preview, but got an error saying I had to leave a comment, so adding it here to...Something happened with the preview of this one - only the code shows. Need to double check what's going on before we push it. The first 2 pages look fine and are ready to go. For this one, maybe it's just the preview that's broken but we need to have a look and see.

**Review by LPipes4 — CHANGES_REQUESTED** [review]

File: common-design-patterns/site-footer.md (hunk 7 above)
Comment (prycrane) [reply common-design-patterns/site-footer.md:1]: Fixed

File: common-design-patterns/site-footer.md
Comment (prycrane) [reply common-design-patterns/site-footer.md:1]: Fixed.

**Review by LPipes4 — APPROVED** [review]

**Comment by prycrane** [comment]
@delisma - please confirm that the h1 will present properly once published.  It's an older pull request that was waiting for https://github.com/canada-ca/systeme-conception/pull/128 to be merged (now merged).  Give it a final check (I removed all the feedback references) and we should be good to go.  @LPipes4 is reviewing the French pull request (https://github.com/canada-ca/systeme-conception/pull/131). Thanks.

**Review by delisma — APPROVED** [review]
LGTM

**Pull request closed without merging**